lib/datadog/ci/contrib/instrumentation.rb

```diff
@@ -18,13 +19,18 @@ def self.register_integration(integration_class)
           @registry[integration_name(integration_class)] = integration_class.new
         end
 
+        def self.auto_instrumented?
+          @auto_instrumented
+        end
+
         # Auto instrumentation of all integrations.
         #
         # Registers a :script_compiled tracepoint to watch for new Ruby files being loaded.
         # On every file load it checks if any of the integrations are patchable now.
         # Only the integrations that are available in the environment are checked.
         def self.auto_instrument
           Datadog.logger.debug("Auto instrumenting all integrations...")
+          @auto_instrumented = true
```

Comment:
Should this be after
```
if auto_instrumented_integrations.empty?
```
since you can end up running without instrumentation?
?